sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,174 @@
+- Add queue requirement when replay a graph [2c53815c6c1e3f8e40c5b6dbe3e5dc2aebd4fa37]
+
+## Bug Fixes
+### SYCL Compiler
+- Fix backend option passing issues with multiple targets [12ca2db0e30b155c7d6bf14320dd1ba468ac9294]
+- Fix -fgpu-rdc option for CUDA [f7595ac7527e67a59e33550851366db7d886ac3f]
+- Fix triple in offload mismatch warning [0fd9a4eb17a2db0fed151ba9d975804322966bcc]
+- Fix a bug when -enable-global-offset=false for AMDGU and NVPTX targets [00cf4c29740b2ec8d027e59e49e376a211599a58]
+
+### SYCL Library
+- Fix stack corruption in sycl::modf for sycl::half type [7c6541dc18e324e36853b473a15e8df4edf97755]
+- Fix nextafter with half on host [1f10f3588e8ccab79e42eee7fd640153d701b08c]
+- Fix identity values for (sub)group collectives min/max [2483893719324831eebce8c188aed688a8a04419]
+- Improve handling of large reqd_work_group_size values [25c3666dffa935c64db513d0db25205e0400c6b7]
+- Fix hip prefetch/mem_advise (noop) for rocm < 5.0 [7e4cc11139871da9e4662e535d4335b823b2118e]
+- Fix ranges to allow default constructor to 0s [0289d2ac94f9692e2067a3c32fb290169d84cbee]
+- Fix propagation of aspects from global variables [fb368a13044897d8ac693f72c442fe1f257c05d2]
+- Missing return statement in the multi_ptr<void..> conversion operator [836469ba88e292ce8dd662074117881c3fdd10d7]
+- Fix `convert` to `sycl::vec<bool, N>` [9dfaf274e2cbe42e992be0519fa7d2b0eb0face5]
+- Ensure queue_impl is not unnecessarily referenced in kernel fusion [a33b0a950ae679423aff505fdbd4a2cae77241eb]
+- Fix int overflow for OffsetInBytes in accessors [2fb0b140ad3be544552a1308fe49ce491bd2881a]
+- Fix invalid discard_write in atomic_memory_order_seq_cst [f69dae7972062d90f804c65996d31f246d2288b3]
+- Fix std::enable_if typo for marray and multi_ptr [0ef25ec3bdf8497dbe272e0295708d704cbe7020]
+- Fix integer type overflow in SYCLDeviceRequirements [cee07d3d26520fcdaab33dfaf7759b59210675d4]
+- Fix UB in group sort scratch memory access [ed4c01301a7c2be9157df6410be3b72ac4b95df7]
+- Use primary context in HIP adapter [d1c92cb95b29b0bf44683cc84b9978d767e0a300]
+- Allow passing default constructed accessors to kernels [fb8ddbf7b0ea3f22b6a2d7fce661e44123ebb2d5]
+- Fix issues with sycl_ext_oneapi_graph subgraphs [92ddf8d562d736f6c556976dce2c099e6aef591e]
+- Fix ambiguous log2 call in host code
+- Add device copyable trait for annotated_arg [9f65e61a2dbf9e9b631159a5ac2ec92137996f62]
+
+### Documentation
+- Update `device_image_scope` property  in `sycl_ext_oneapi_device_global` [be8e031cd078ea0298a773aa5fc743c18e50fe97]
+
+## API/ABI breakages
+- Deprecate experimental functions: quot,div,mod,dp*,line,lrp [76976a22ba2e86059d655aa7d5f71250160f4864]
+- Remove ESIMD Emulator plugin [1ece6da6b1139ad29d1ed44a05759e9b5974d3e8]
+- Deprecate `sycl::abs(genfloat)` [d15d4409b2cf7e023bf6f462fac5630699f1c0a7]
+- Don't include `<complex>/<cmath>` from `sycl.hpp`, unless `-fpreview-breaking-changes` [6ed0ab887d18b18209a7d1c31b8bb8240a820f79] [d22c9c58d2e7c4936158dde6056281ef2cf0cdf2]
```

Comment:
Tagging @aelovikov-intel and @steffenlarsen here for suggestions. Strictly speaking, this change didn't happen, because it is hidden under a flag. Should we somehow specially annotate such changes as "future API/ABI breackages"?

In any case, "unless `-fpreview-breaking-changes`" -> "if `-fpreview-breaking-changes` is set"

Reply:
I do think it would be best to mark such changes as being behind the preview flag somehow.

Reply:
We had an approval just to drop the includes unconditionally. I just didn't push through with https://github.com/intel/llvm/pull/11528.